dependencies.gradle

```diff
@@ -8,7 +8,7 @@ project.ext.versions = [
     opencsv: '2.3',
     gson: '2.2.4',
     http: '4.5.1',
-    jts: '1.18.0',
+    jts: '1.18.2-SNAPSHOT',
```

Comment:
Will this be updated to a release when ready to merge?

Reply:
No. It's a significant step back in performance (as in, on a scale of over 400x slower). It doesn't change enough for this snapshot to be a concern.

Reply:
looks like JTS is doing a release in a few weeks, so this won't be a huge deal either way. it'll either release in time for this PR to get updated, or we'll release with snapshot and upgrade ASAP shortly after.